Units/parser-cxx.r/bug1575055.cpp.d/expected.tags

```diff
@@ -1,3 +1,3 @@
 MyClass	input.cpp	/^	class MyClass { };$/;"	c	namespace:TheNamespace	file:
 TheNamespace	input.cpp	/^namespace TheNamespace {$/;"	n	file:
-variable	input.cpp	/^	int variable;$/;"	m	namespace:TheNamespace	file:
+variable	input.cpp	/^	int variable;$/;"	v	namespace:TheNamespace
```

Comment:
v is correct as per our convention. m are members of class/struct/union only.
(though the C++ spec talks also about namespace members and since global variables are in the global namespace then there would be no v variables at all... :D )

Again the same policy.. I've though about the symbol itself and not really the declaration being potentially visible from other files. Maybe my idea is wrong. I can change that.